aroma/tests/data/classification_overview.txt

```diff
@@ -1,45 +1,5 @@
-IC	Motion/noise	maximum RP correlation	Edge-fraction	High-frequency content	CSF-fraction
-1	True	0.65	0.65	0.96	0.00
-2	True	0.86	0.66	0.08	0.00
-3	True	0.89	0.62	0.13	0.01
-4	True	0.62	0.80	0.96	0.00
-5	False	0.54	0.50	0.09	0.01
-6	True	0.44	0.72	0.05	0.00
-7	True	0.42	0.83	0.94	0.00
-8	False	0.26	0.42	0.09	0.02
-9	False	0.78	0.45	0.13	0.00
-10	False	0.22	0.45	0.09	0.04
-11	False	0.32	0.62	0.11	0.01
-12	False	0.34	0.51	0.09	0.01
-13	True	0.39	0.59	0.58	0.01
-14	False	0.39	0.60	0.13	0.00
-15	True	0.71	0.69	0.90	0.00
-16	False	0.34	0.57	0.09	0.01
-17	False	0.54	0.55	0.21	0.01
-18	False	0.40	0.40	0.03	0.00
-19	False	0.28	0.61	0.16	0.00
-20	False	0.39	0.48	0.08	0.00
-21	False	0.16	0.60	0.11	0.02
-22	True	0.58	0.79	0.81	0.01
-23	False	0.30	0.43	0.13	0.00
-24	True	0.59	0.60	0.91	0.01
-25	False	0.21	0.33	0.31	0.00
-26	False	0.46	0.55	0.23	0.01
-27	False	0.32	0.29	0.14	0.00
-28	False	0.53	0.55	0.11	0.01
-29	False	0.19	0.72	0.13	0.00
-30	False	0.34	0.56	0.28	0.05
-31	False	0.42	0.60	0.18	0.02
-32	False	0.22	0.23	0.17	0.05
-33	False	0.33	0.53	0.22	0.02
-34	False	0.29	0.54	0.27	0.01
-35	False	0.31	0.44	0.16	0.01
-36	False	0.34	0.50	0.18	0.02
-37	False	0.38	0.51	0.17	0.02
-38	False	0.40	0.53	0.21	0.03
-39	False	0.30	0.60	0.22	0.02
-40	False	0.28	0.48	0.23	0.02
-41	False	0.45	0.47	0.23	0.01
-42	False	0.33	0.48	0.29	0.04
-43	False	0.29	0.37	0.16	0.04
-44	False	0.47	0.58	0.24	0.01
+IC	edge_fract	csf_fract	max_RP_corr	HFC	classification
+0	0.6673487194914882	0.004789119897316091	0.6239073998302727	0.9627976190476192	rejected
+1	0.6560402271978647	0.004793030026303001	0.8509440133229171	0.08234126984126983	rejected
+2	0.8037272625459998	0.0031916636290894155	0.6323438789815838	0.9627976190476192	rejected
+3	0.623519424401591	0.005318133138776824	0.8048700572409935	0.09474206349206347	rejected
```

Comment:
Why does the new file only have 4 components?

And why does it start from 0?

Reply:
This is literally the output. Could you pull the PR and try at your computer?

Reply:
OHHHHHH, i think i know what's happening.

Reply:
The result has more components. However you are only checking the first four: 
```
    true_classification_overview = pd.read_csv(join(resources_path, "classification_overview.txt"),
                                               sep="\t",
                                               index_col="IC",
                                               nrows=4,)
    classification_overview = pd.read_csv(
        join(out_path, "classification_overview.txt"), sep="\t", index_col="IC", nrows=4
    )
```
Look at the nrows=4 argument. So we could have all the components storaged in the repo but we are only using the 4 firsts for checking

Reply:
We should have all of them saved. Then we get to choose how many we want to actually compare with.

Reply:
Fair enough

Reply:
let me update then

Reply:
done